modules/markdown/sanitizer_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Copyright 2017 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package markdown
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
goconvey has been removed before. I think change this test file to `github.com/stretchr/testify` is better